PSCompatibilityAnalyzer/Microsoft.PowerShell.CrossCompatibility/Query/Modules/ModuleData.cs

```diff
@@ -51,12 +57,12 @@ public ModuleData(string name, Version version, Data.Modules.ModuleData moduleDa
         /// <summary>
         /// Functions exported by the module.
         /// </summary>
-        public IReadOnlyDictionary<string, FunctionData> Functions => _commands.Value.Item1;
+        public IReadOnlyDictionary<string, FunctionData> Functions => _lazyCommands.Value.Item1;
```

Comment:
C# 7.1 introduced named Tuples, see docs [here](https://devblogs.microsoft.com/dotnet/new-features-in-c-7-0/). It would be great if you could use them for better readability (although strictly speaking this is something that your PR does not introduce, so happy to have a a 2nd PR to apply this later.

Reply:
I prefer them too but was working on the impression that those use `ValueTuple` rather than `Tuple`. `ValueTuple` requires an extra dependency, which I wanted to avoid for now. Would be happy to introduce later.

Reply:
Ok. Maybe a separate PR then? We already have a reference to a C Sharp NuGet package. I think we would only need to add the reference but not ship the dll.

Reply:
[Not in 452](https://docs.microsoft.com/en-us/dotnet/api/system.valuetuple-4?view=netframework-4.7.2&viewFallbackFrom=netframework-4.5.2) apparently. We'd need to ship a DLL for it.

Reply:
Ok, might be not worth the effort of adding more logic to the build process then.